content/docs/user-guide/gems/reference/aws/aws-gamelift/cpp-api.md

```diff
@@ -1,19 +1,20 @@
 ---
 linkTitle: C++ API
-title: C++ API for AWS GameLift Gem
-description: "Learn how to use the C++ API for AWS GameLift Gem in O3DE"
+title: AWS GameLift Gem C++ API 
+description: Learn how to use the C++ API for the AWS GameLift Gem in Open 3D Engine (O3DE).
 toc: true
 ---
 
-## 1. Session Interface
+## Session interface
 
-The *session interface*, `ISessionRequests` and `ISessionAsyncRequests`, provides public APIs that allows you to create game sessions and allows players to search and join online games. The session interface abstracts the implementation details of session related management.
+The **AWS GameLift** Gem implements a session interface for Amazon's dedicated server solution, Amazon GameLift. The *session interface* (`ISessionRequests` and `ISessionAsyncRequests`) provides public APIs that allow you to create game sessions and allow players to search and join online games. The session interface abstracts the implementation details of session-related management.
+
+The session interface performs all of the session handling. The Gem acts as a game-specific handler for the session interface. The game code makes calls by using the Gem's C++ API to interact with the session. GameLift creates and owns the game session, which exists on the server only when running an online game.
 
 There must be only one implementation of the session interface per dedicated server solution. To add support for another dedicated server solution, you must create another implementation of the session interface.
 
-The AWS GameLift Gem implements a session interface for Amazon's dedicated server solution, Amazon GameLift service. The session interface performs all of the session handling, the AWS GameLift Gem acts as a game-specific handler of the session interface, and the game code makes calls to it when it needs to interact with the session. The game session is created and owned by Amazon GameLift service, and only exists on the server when running an online game.
 
-## 2. Session Management
+## Session management
 
```

Comment:
There is a small update which corresponds to scripting configuregameliftclient as well. https://wiki.agscollab.com/pages/viewpage.action?pageId=126422034#C++APIforAWSGameLiftGem-SessionManagement

like what we have talked, if customer can do it in scripting, then customer can do it through c++ as well. ScriptCanvas node is just a reflection of c++ api, node is triggering c++ api under the hood.